src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -305,18 +443,28 @@ protected override ImmutableArray<PendingBranch> Scan(ref bool badRegion)
             MethodSymbol methodSignatureOpt,
             ArrayBuilder<(BoundReturnStatement, TypeSymbolWithAnnotations)> returnTypes,
             VariableState initialState,
-            Action<BoundExpression, TypeSymbolWithAnnotations> callbackOpt)
+            PooledDictionary<BoundExpression, (NullabilityInfo, TypeSymbol)> analyzedNullabilityMapOpt = null)
```

Comment:
> = null [](start = 102, length = 7)

Are there any callers that use the optional value? If not, please remove.

Reply:
Are you asking if there are any callers that don't pass along a map? If that's it, then yes, there's 1. I can make them pass a value for it instead if you'd prefer.

---
In reply to: [265689134](https://github.com/dotnet/roslyn/pull/34017#discussion_r265689134) [](ancestors = 265689134)

Reply:
Yes, if there's only one caller that uses the optional value, let's make the parameter required, especially since this is a `private` method.

---
In reply to: [265694467](https://github.com/dotnet/roslyn/pull/34017#discussion_r265694467) [](ancestors = 265694467,265689134)